packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -239,10 +244,12 @@ const AnimatedContainer = ( {
 	);
 };
 
-const BlockInspectorSingleBlock = ( { clientId, blockName } ) => {
+const BlockInspectorSingleBlock = ( { clientId, blockName, block } ) => {
 	const availableTabs = useInspectorControlsTabs( blockName );
 	const showTabs = availableTabs?.length > 1;
-
+	const hasBindings = block?.attributes?.metadata?.bindings
+		? Object.keys( block.attributes.metadata.bindings ).length > 0
+		: false;
```

Comment:
Should we put this check for the bindings here, as well as in SettingsTab, or should all of this logic be contained inside of the Block Bindings Panel?